Adding python cassandra-driver

jaustinpage opened the pull request.

jaustinpage commented on testing/py-cassandra-driver/APKBUILD:
```diff
@@ -0,0 +1,50 @@
+pkgrel=0
+pkgdesc="Python driver for Apache Cassandra"
+url="https://pypi.python.org/pypi/cassandra-driver/"
+arch="noarch"
```
Im not sure if this should be noarch or all...

jirutka commented on testing/py-cassandra-driver/APKBUILD:
```diff
@@ -0,0 +1,50 @@
+	_py python2
+	replaces="$pkgname"
+	depends=""
+}
```
This is wrong, please follow [example on the wiki](https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python).

jaustinpage commented:
Updated this to match the wiki example, and switched it from building against the github package to building against pypi for more consistency in upgrades in the future.
